### ISSUE TITLE
Keyboard focus is not moving to the ﬁrst interactive control after invoking the connections Tab item in the left menu:A11y_SQL Azure Data Studio_Create new database project_Keyboard

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#26025]
Please do not close this bug. This bug should only be closed by Trusted Tester after veriﬁcation 
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug ﬁxing faster and easier." 

### GitHub Tags:
#A11yWCAG; #A11ySev3; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Nov2024; #SQL Azure Data Studio; #WCAG2.4.3;  #Win32; #Benchmark; #FTP; #STP;  #FocusOrder;#Keyboard;

### Environment Details:
Application Name: SǪL Azure Data Studio 
Edition: Windows 11 Enterprise 
Version: 24H2 
Installed: 22-10-2024 
OS build: 26100.2033 
Experience: Windows Feature Experience Pack 1000.26100.23.0 

### Steps to Reproduce
1.Launch the application-Azure Database Project and install extension SǪL Database Projects. 
2.Tab to “Database projects” tab item present in the left menu and Hit ENTER key. 
3.Observe whether after pressing Tab key, keyboard focus is moving to the ﬁrst interactive control in the connections section or not. 

### Actual Results:
Keyboard focus order is not correct after invoking “Connections” Tab item in the left menu. Keyboard focus is not moving to the ﬁrst interactive control in the Connections section and it is moving to the Accounts tab and with next tab keyboard focus to the moving to the Connections section. 

### Expected Results:
Keyboard focus order should be correct after invoking “Connections” Tab item in the left menu. Keyboard focus should move to the ﬁrst interactive control in the Connections section. 

### 
### User Impact:
Keyboard only users will face difficulty while navigating through the page if focus order will be incorrect.
